bundles/org.openhab.binding.jeelink/src/main/resources/OH-INF/i18n/jeelink_de.properties

```diff
@@ -19,6 +19,8 @@ thing-type.ec3k.label = ec3k
 thing-type.ec3k.description = Ein mit dem JeeLink USB Empfänger verbundenes EC3000 Energiekosten-Messgerät.
 thing-type.pca301.label = PCA301
 thing-type.pca301.description = Eine mit dem JeeLink USB Empfänger verbundene PCA301 Steckdose.
+thing-type.emt7110.label = EMT7110
+thing-type.emt7110.description = Eine mit dem JeeLink USB Empfänger verbundene EMT7110 Steckdose.
```

Comment:
Please revert changes as only the default language is supplied by github, other languages are done by crowdin.